cypress/integration/newsBodySpec.js

```diff
@@ -86,7 +87,7 @@ describe('Article Body Tests', () => {
     );
   });
 
-  it('should have a working first inline link', () => {
-    clickInlineLinkAndTestPageHasHTML('main a', '/news/articles/c85pqyj5m2ko');
-  });
+  // it('should have a working first inline link', () => {
```

Comment:
Instead of commenting out this test, I'd suggest changing the assertion, so we continue to test the first inline link of a page.